### DISCUSSION TITLE
Working on a Dropbox file finder plugin

### DISCUSSION THREAD

**Discussion by ameenaltajer** [Flow-Launcher/Flow.Launcher#1148] [Ideas]
Hello guys,

I've been very interested in having a Dropbox file finder plugin for Microsoft PowerToys Run, however since the plugin SDK/workflow is still now done I found out that Flow Launcher is far more mature, and to my luck, it has the SDK/Samples are done nicely, so I figured I build it for Flow Launcher since I switched all together.

I'm working on the plugin here, works for now but is still in a work in progress state with few hardcoded values until I write a settings menu later on after I finish the Dropbox integration:

https://github.com/ameenaltajer/FlowLauncher.DropboxFinder

Just keeping you all posted on this, as somebody might have the same interest and send a couple of pull requests, thanks for a great project guys.

Best,
Ameen

**Comment by taooceros** [comment]
Hi Ameen, It's nice to see that you like Flow! Why not submit a pull request on the plugin manifest https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest/pulls so that others can download it easily?

**Reply by ameenaltajer** [reply]
Sure thing, I'll just finish the OAuth flow and do that :)

Thanks!

**Reply by ameenaltajer** [reply]
All done, and submitted the PR! Thanks for a great launcher guys! Hope people benefit out of this :)

Ameen
